seedVault/encryptionUtils.ts

```diff
@@ -0,0 +1,106 @@
+import { mnemonicToEntropy, wordlists } from 'bip39';
+
+interface EncryptMnemonicArgs {
+  password: string;
+  seed: string;
+  mnemonicEncryptionHandler: (seed: string, key: string) => Promise<Buffer>;
+}
+
+export async function encryptMnemonicWithHandler(cb: EncryptMnemonicArgs) {
+  const { mnemonicEncryptionHandler, password, seed } = cb;
+  const encryptedSeed = await mnemonicEncryptionHandler(seed, password);
+  return encryptedSeed.toString('hex');
+}
+
+interface DecryptMnemonicArgs {
+  password: string;
+  encryptedSeed: string;
+  mnemonicDecryptionHandler: (seed: Buffer | string, key: string) => Promise<string>;
+}
+
+export async function decryptMnemonicWithHandler(cb: DecryptMnemonicArgs) {
+  const { mnemonicDecryptionHandler, password, encryptedSeed } = cb;
+  const seedPhrase = await mnemonicDecryptionHandler(encryptedSeed, password);
+  return seedPhrase;
+}
```

Comment:
I'm not sure how these relate to the seed vault. if they are related, could we make their relationship to the seed vault clear with some sort of typing? If they aren't directly related, maybe they should go back to the wallet section if it's a more appropriate location.